coral-component-shell/src/scripts/ShellMenuBarItem.js

```diff
@@ -418,6 +418,10 @@ class ShellMenuBarItem extends BaseComponent(HTMLElement) {
     if (this.menu !== null) {
       this.menu = this.menu;
     }
+
+    if (this._elements.shellMenuButton.getElementsByTagName('coral-icon')[0]) {
```

Comment:
Please store the shellMenuButton icon in a variable. 
You can use querySelector instead of getElementsByTagName

this._elements.shellMenuButton.querySelector("coral-icon')

Reply:
moved changes to https://github.com/adobe/coral-spectrum/pull/342